### ISSUE TITLE
Return "host"

### ISSUE THREAD

**Issue by theDanielJLewis** [fancysoups/podcast-privacy#7]
It'd be great if your script could also return the actual host of the media file.

For example, if someone is using Chartable ➜ Blubrry ➜ Libsyn, you would be able to see that Libsyn is the host. But if you saw something like Podtrac ➜ Blubrry, then Blubrry would be the host.

It might be as simple as picking the _last_ path in the URL, or related to #1, you might have to follow the redirects.

**Comment by shahruz** [comment]
@theDanielJLewis I had something similar integrated in an earlier pre-release version, with "Host" listed as an ability, but opted to remove it since it felt out of scope for informing listeners about privacy-related issues. 

I'd be happy to reconsider that if it feels like that information would be useful.

**Comment by theDanielJLewis** [comment]
Yeah, I understand. Although it might seem to fit better under a name like "Podcast Info," I think knowing where the media is hosted could still be of interest for privacy reasons.

For example, it's primarily the _host_ that can track streaming progress, not so much a standard redirect prefix.

Special care would need to be used for providers like Blubrry where they can be either host+stats or stats-only. But you could learn that from the enclosure URL.
